runtime/gc_structs/ClassLoaderClassesIterator.cpp

```diff
@@ -47,15 +47,7 @@ GC_ClassLoaderClassesIterator::GC_ClassLoaderClassesIterator(MM_GCExtensionsBase
 J9Class *
 GC_ClassLoaderClassesIterator::nextSystemClass() 
 {
-	J9Class * result = NULL;
-	J9Class ** slot = NULL;
-	while (NULL != (slot = _vmClassSlotIterator.nextSlot())) {
-		result = *slot;
-		if (NULL != result) {
-			break;
-		}
-	}
-	return result;
+	return _vmClassSlotIterator.nextSlot();
```

Comment:
@dmitripivkine Can you double check that this is indeed correct?

Reply:
yes, this looks correct